db/Models/news.models.js

```diff
@@ -0,0 +1,18 @@
+const db = require("../connection");
+const format = require("pg-format");
+
+const selectTopics = () => {
+  let queryString = "SELECT * FROM topics";
+
+  return db.query(queryString).then((data) => {
+    if (data.rows.length === 0) {
+      return Promise.reject({ status: 400, msg: "Bad request" });
```

Comment:
This case of nothing coming back to the model would be more of a 404 - just because we have no topics doesn't necessarily mean that a bad request has been made.

A bad request is more along the lines of an incorrect data type in the request the user has made. Our server having no results come back to the model is instead a 404.

Also, we can't really test for this as the test database will always have a topic present. We can safely remove this couple of lines